.github/workflows/pull_request.yml

```diff
@@ -22,3 +22,26 @@ jobs:
 
       - name: Run the validations
         run: npm run validate
+
+  check_for_changed_files:
```

Comment:
Can't a release be after a few PR's then we decide to "seal" the version?
I feel like this is too strict.

Reply:
it compares to the last release, so how is it strict?

Reply:
I mean that it seems to not allow changes in schemas unless it's part of a new release, but maybe we should allow this in order to let people make changes until there's an agreed upon version.